cmd/api/auth/authn_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package auth
+
+import (
+	"context"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	apisMeta "k8s.io/apimachinery/pkg/apis/meta/v1"
```

I think you can drop the "v1" from this abbreviation. I doubt we will be using v1 and v2 at the same time. But, you should add "apis" to make it more specific than "meta": I suggest "apisMeta".

Reply:
Solving it to keep with the standard practice